CHANGELOG.md

```diff
@@ -1,24 +1,41 @@
 # Change Log
 
 ## TODO: ideas, issues and planed extensions or changes that are not yet implemented
+
 * optimize add_qc_metrics for run after new samples have been added - should not recompute everything
-* planned new feature: during import of long reads, (optionally) correct for short exon alignment issues. 
+* planned new feature: during import of long reads, (optionally) correct for short exon alignment issues.
 * separate new read import and classification of isoforms.
 
+## [2.0.0]
+
+* supported the analysis of Oxford Nanopore data
+* new option TSS identification according to the reference annotation
+* new gene model characteristics, simplex coordinates and relative entropy
+* new visualisation using triangle plot
+* fixed bugs in external gtf import to reconstruct transcriptome
+* imporoved the coordination analysis of TSS and PAS
+* supported the import of SQANTI QC report and filtering based on it
+* supported the export of positive and negative TSS for ML
+* supported proteogenomic approaches at the interface of transcriptomics and proteomics
+* improved code readability and filter tag construction
+* ...
```

Comment:
I wouldn't write 'support**ed**'  as it sounds as if support stopped. Instead use something like 'added support for'